src/test/java/com/microsoft/sqlserver/jdbc/clientcertauth/ClientCertificateAuthenticationTest.java

```diff
@@ -2,7 +2,7 @@
  * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
  * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
  */
-package com.microsoft.sqlserver.clientcertauth;
+package com.microsoft.sqlserver.jdbc.clientcertauth;

```

Comment:
This line is already covered in PR 1313, you should remove this in one of them (probably in the other one)